dist/style/roadmap.css

```diff
@@ -0,0 +1,34 @@
+.roadmap-container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 20px;
+    justify-content: center;
+}
+
+.roadmap-card {
+    background-color: white;
```

Comment:
The white background looks kind of out of place to me, perhaps going for dark blue with white text is better?

Reply:
Will give it a try o7

Although I guess at this point this PR has been superseded by #75 lol